src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -950,9 +949,12 @@ bool FancyZones::IsNewWorkArea(GUID virtualDesktopId, HMONITOR monitor) noexcept
 
 bool FancyZones::IsSplashScreen(HWND window)
 {
-    TCHAR splashString[] = L"MsoSplash";
-    TCHAR className[MAX_PATH];
-    GetClassName(window, className, _countof(className));
+    wchar_t splashString[] = L"MsoSplash";
+    wchar_t className[MAX_PATH];
+    if (GetClassName(window, className, wcslen(className)) == 0)

```

Comment:
this isn't correct either, because now we're passing the length of a `className` buffer by reading garbage from uninitialized `className` buffer. we could just pass `MAX_PATH` here.